xapian-applications/omega/query.cc

```diff
@@ -1116,6 +1117,7 @@ T(relevant,	   0, 1, N, Q), // is document relevant?
 T(relevants,	   0, 0, N, Q), // return list of relevant documents
 T(score,	   0, 0, N, 0), // score (0-10) of current hit
 T(set,		   2, 2, N, 0), // set option value
+T(seterror,	   1, 1, 1, 0), // set error_msg, setting it early stops query execution
```

Comment:
Make the 4th parameter `N` rather than `1`, like we do for all the existing commands which evaluate all their arguments (the processing code doesn't care if evalargs > maxargs, and by using `N` we don't have the potential trap of someone adding a new optional argument to a command and increasing maxargs, but failing to also increase evalargs).